templates/BUILD.bazel

```diff
@@ -24,6 +28,12 @@ genrule(
         rm $$OUT/quickstart-java/da-skeleton.yaml
         cp -rL $$SRC/quickstart-java/* $$OUT/quickstart-java/
 
+        # quickstart-scala template
+        mkdir -p $$OUT/quickstart-scala
+        cp -r $(location //language-support/scala/examples:quickstart-scala-dir)/* $$OUT/quickstart-scala/
+        rm $$OUT/quickstart-scala/da-skeleton.yaml
```

Comment:
Do we really need to have a `da-skeleton.yaml` file in this template? I don’t see us adding it to the old assistant ever.